third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.h

```diff
@@ -38,6 +43,30 @@ namespace google_smart_card {
 // already been initialized.
 void InitializeAndRunPcscLiteServer();
 
+class PcscLiteServerGlobal final {
```

Comment:
Please write, at least short, comment with the class description.
Also it makes sense to mention the thread safety principles used in this class (e.g. that most methods are generally thread-safe, except GetInstance which is not protected against concurrent class construction or destruction).